backend/core/serializers/user_settings.py

```diff
@@ -86,6 +86,8 @@ class VideosPollUserSettingsSerializer(GenericPollUserSettingsSerializer):
         ("ALL_TIME", "all_time"),
     ]
 
+    extension__search_reco = serializers.BooleanField(required=False)
```

Comment:
Maybe we could add more context about this setting in a help text:

```suggestion
    extension__search_reco = serializers.BooleanField(
        required=False,
        help_text=(
            "Whether Tournesol recommendations should be integrated in Youtube.com search results."
        ),
    )
```